sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/engine/RealSyncEngine.kt

```diff
@@ -232,6 +237,50 @@ class RealSyncEngine @Inject constructor(
         }
     }
 
+    private fun getDeletionRequests(): List<Pair<SyncDeletionRequest, DeletableDataManager>> {
```

Comment:
I’m not a fan of returning Pairs from a function. You can simplify how deletions are processed all in one function instead. I’m guessing you are doing it this way because you want to filter deletions, but that shouldn’t be necessary.